app/javascript/components/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Greeting from "./Greeting";
+import configureStore from "../configureStore";
+import { Provider } from "react-redux";
+const store = configureStore();
+
+class App extends React.Component {
+  render () {
+    return (
+      <Provider store={store}>
+        <React.Fragment>
+          <Router>
+          <Routes>
+            <Route path="/" element= { ("Home!") }/>
+            <Route path="/hello" element={ <Greeting greeting="Friend" /> } />
```

Comment:
## OPTIONAL 
* I am unable to display random greetings even changing the API URL to my localhost ` http://localhost:3000/v1/greetings` am making this an optional suggestion because it wasn't mentioned by the previous reviewer, but i strongly suggest you fix this as the aim of this project is to Implement a connection between a Ruby on Rails backend and React front-end.